### PR TITLE
Added reset-graphics-mode code to strip_ansi_ecsape_codes list. Added…

### PR DESCRIPTION
**Pull request by mreams13**
… fix to strip ANSI escape codes before prompt search in send_command.

### CODE REVIEW

File: netmiko/base_connection.py [hunk 1]
```diff
@@ -1354,7 +1357,7 @@ def strip_ansi_escape_codes(self, string_buffer):
         code_set = [code_position_cursor, code_show_cursor, code_erase_line, code_enable_scroll,
                     code_erase_start_line, code_form_feed, code_carriage_return,
                     code_disable_line_wrapping, code_erase_line_end,
-                    code_reset_mode_screen_options, code_erase_display,
+                    code_reset_mode_screen_options, code_reset_graphics_mode, code_erase_display,
```
Comment (ktbyers) [inline netmiko/base_connection.py]: Yes, this looks totally fine.

File: netmiko/base_connection.py [hunk 2]
```diff
@@ -1028,6 +1028,8 @@ def send_command(self, command_string, expect_string=None,
         while i <= max_loops:
             new_data = self.read_channel()
             if new_data:
+                if self.ansi_escape_codes:
+                    new_data = self.strip_ansi_escape_codes(new_data)
```
Comment (ktbyers) [inline netmiko/base_connection.py]: I am a bit worried about how slow this is going to be...I assume this was on some Linux platform (I seem to recall on Slack or on your GitHub issue that you had stated it was)?

Comment (mreams13) [reply netmiko/base_connection.py]: I have to use Windows 10 for my development machine.

Comment (ktbyers) [reply netmiko/base_connection.py]: This is probably fine...let me just see how long this takes to execute, but it is probably no issue.

**Pull request merged**